SU-SWS/stanford_basic
# D8CORE-2268: adjusting the spacing for media files in the wysiwyg and fixing the centering for the pdf.

# READY FOR REVIEW

# Summary
- Fixing the alignment of media that are files added to the WYSIWYG

# Review By (Date)
- 12/3

# Urgency
- Med

# Steps to Test

1. Pull in change.
2. Open a page and edit text. 
3. Embed a media file. Use the "Edit Media" to align one left, one right, one centered and one with none. 
4. Verify the items are aligned with padding on the approprieate sides.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- D8CORE-2268
- @cjwest 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
